lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -61,62 +81,86 @@ export class ScopeSet {
         this.appendScope(Constants.OFFLINE_ACCESS_SCOPE);
     }
 
+    /**
+     * Replace client id with the default scopes used for token acquisition.
+     */
+    private addDefaultScopes(): void {
+        this.appendScope(Constants.OPENID_SCOPE);
+        this.appendScope(Constants.PROFILE_SCOPE);
+        this.appendScope(Constants.OFFLINE_ACCESS_SCOPE);
+    }
+
     /**
      * Used to validate the scopes input parameter requested  by the developer.
      * @param {Array<string>} inputScopes - Developer requested permissions. Not all scopes are guaranteed to be included in the access token returned.
      * @param {boolean} scopesRequired - Boolean indicating whether the scopes array is required or not
-     */
+    */
     private validateInputScopes(inputScopes: Array<string>): void {
         if (this.scopesRequired) {
             // Check if scopes are required but not given or is an empty array
             if (!inputScopes || inputScopes.length < 1) {
-                throw ClientConfigurationError.createEmptyScopesArrayError(inputScopes);
+                throw ClientConfigurationError.createEmptyScopesArrayError(
+                    inputScopes
+                );
             }
         }
 
         // Check that scopes is an array object
         if (!Array.isArray(inputScopes)) {
-            throw ClientConfigurationError.createScopesNonArrayError(inputScopes);
+            throw ClientConfigurationError.createScopesNonArrayError(
+                inputScopes
+            );
         }
     }
 
     /**
      * Check if a given scope is present in this set of scopes.
-     * @param scope 
+     * @param scope
      */
     containsScope(scope: string): boolean {
-        return !StringUtils.isEmpty(scope) ? this.scopes.has(scope) : false;
+        return !StringUtils.isEmpty(scope)
+            ? this.scopes.has(scope)
+            : false;
```

Comment:
same argument about readability

Reply:
auto correct by "prettify". We can change the linter rules as we see it before final merge.

Reply:
Edited.